models/zeroex/polygon/zeroex_polygon_native_fills.sql

```diff
@@ -5,15 +5,15 @@
         unique_key = ['block_date', 'tx_hash', 'evt_index'],
         on_schema_change='sync_all_columns',
         file_format ='delta',
-        incremental_strategy='merge' 
+        incremental_strategy='merge'
```

Comment:
looks like the target one is missing?

Reply:
yes thanks for catching, I'll follow up with another PR